config/rootfs/debos/scripts/buster-ltp.sh

```diff
@@ -48,6 +48,11 @@ cd testcases/open_posix_testsuite/ && make generate-makefiles
 make all
 mkdir /opt/openposix_testsuite/bin -p
 make install
+
+# Make sure openposix_testsuite placed under /opt/ltp
```

Comment:
Can we not tell `make install` to install files directly in `/opt/ltp` instead of moving them?

Reply:
From the Makefile:

```
prefix?=                `$(top_srcdir)/scripts/print_prefix.sh`
```

Reply:
So here's a better way to fix it:
```
$ make generate-makefiles 
$ grep prefix ./conformance/behavior/timers/Makefile
prefix?=		/opt/openposix_testsuite
exec_prefix?=		$(prefix)
INSTALL_DIR=		$(DESTDIR)/$(exec_prefix)/$(subdir)
$ make generate-makefiles prefix=/opt/ltp
$ grep prefix ./conformance/behavior/timers/Makefile
prefix?=		/opt/ltp
exec_prefix?=		$(prefix)
INSTALL_DIR=		$(DESTDIR)/$(exec_prefix)/$(subdir)
```

Reply:
So how about replacing:
```
cd testcases/open_posix_testsuite/ && make generate-makefiles
```
with this?
```
cd testcases/open_posix_testsuite/
make generate-makefiles prefix=/opt/ltp
```

Reply:
```
make generate-makefiles prefix=/opt/ltp
make all &&  make install
```
still tries to use different location.
```
install: cannot create regular file '//opt/openposix_testsuite/bin/.': No such file or directory
```

Reply:
Interesting, do you know where the `/opt/openposix_testsuite` path is coming from if it's not in the generated Makefiles?

Reply:
Seems like its hard-coded here? 

```
$cat testcases/open_posix_testsuite/scripts/print-prefix.sh 
#!/bin/sh

if uname -a | grep -iq linux
then
	DEFAULT_PREFIX=/opt
else
	DEFAULT_PREFIX=/usr/local
fi

echo ${prefix:=$DEFAULT_PREFIX/openposix_testsuite}
```

Reply:
No, because if you pass the `prefix=/opt/ltp` variable then it overrides it (that's the `?=` syntax):
```
prefix?=                `$(top_srcdir)/scripts/print_prefix.sh`
```

Maybe you need to keep passing it to `make` and `make install`.

Reply:
right. seems like `make all &&  make install prefix=/opt/ltp`  does the job. Let me check further and update the PR

Reply:
Updated PR: 
https://lava.collabora.co.uk/scheduler/job/4286749
https://lava.collabora.co.uk/scheduler/job/4286750